src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionIntroViewModel.swift

```diff
@@ -13,18 +13,21 @@ class ExposureSubmissionIntroViewModel {
 	init(
 		onQRCodeButtonTap: @escaping () -> Void,
 		onTANButtonTap: @escaping () -> Void,
-		onHotlineButtonTap: @escaping () -> Void
+		onHotlineButtonTap: @escaping () -> Void,
+		onThankYouScreen: @escaping () -> Void
 	) {
 		self.onQRCodeButtonTap = onQRCodeButtonTap
 		self.onTANButtonTap = onTANButtonTap
 		self.onHotlineButtonTap = onHotlineButtonTap
+		self.onThankYouScreen = onThankYouScreen
 	}
 	
 	// MARK: - Internal
 	
 	let onQRCodeButtonTap: () -> Void
 	let onTANButtonTap: () -> Void
 	let onHotlineButtonTap: () -> Void
+	let onThankYouScreen: () -> Void
```

Comment:
I can't see this one get's used? If it is useless, please remove

Reply:
Removed.